src/lib/helpers.ts

```diff
@@ -4,3 +4,6 @@ export const processFile = (
   file === null || typeof file === 'string' ? false : file
 
 export const stopPropagation = (event: Event): void => event.stopPropagation()
+
+export const removeFilename = (path: string): string =>
+  /(.+)(\/.+)$/.exec(path)?.[1] || path
```

Comment:
I would have done something like this: 
- we have the filename 
- we have the fullpath containing the filename

I'd have only do something like filePath.replace(`${filename}$`, '') no? It seems easier to read and understand.

Reply:
Yeah, that also works but I wanted to keep props/args at a minimum level. In terms of performance, I'm not sure if there's a big difference between the two approaches. But in terms of API, I think it's nice to have a single arg function.